example/example-model/src/main/java/uk/gov/gchq/palisade/example/config/ProxyServicesFactory.java

```diff
@@ -108,7 +108,7 @@ public CacheService createInternalCacheService() {
     }
 
     public AuditService createInternalAuditService() {
-        return new LoggerAuditService();
+        return new StroomAuditService().organisation("ORG").systemEnv("Test").systemVersion("v0").systemName("SystemX").systemClassification("Sensitive").systemDescription("A test system");
```

Comment:
I also had that dilemma, I came to the conclusion that all this really does is means that the audit logs will appear in an XML format that Stroom uses (but is relatively readable) rather than as simple lines of text.
As such I am happy to change it back if the consensus is that the LoggerAuditService is the best one to use by default.